cmd/query/app/querysvc/query_service_options.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package querysvc
+
+import (
+	"github.com/jaegertracing/jaeger/model/adjuster"
+	"github.com/jaegertracing/jaeger/storage/spanstore"
+)
+
+// QueryServiceOption is a function that sets some option on the QueryService
+type QueryServiceOption func(qsvc *QueryService)
+
+// QueryServiceOptions is a factory for all available QueryServiceOptions
+var QueryServiceOptions queryServiceOptions
+
+type queryServiceOptions struct{}
+
+// Adjusters creates a QueryServiceOption that initializes the sequence of Adjusters on the QueryService.
+func (queryServiceOptions) Adjusters(adjusters ...adjuster.Adjuster) QueryServiceOption {
```

Comment:
Sorry, I should've been more explicit - let's not use the functional options pattern, we're trying to move away from it, just define the additional fields in QueryServiceOptions struct.